dev/diffs/3.4.2.diff

```diff
@@ -1371,11 +1323,20 @@ index dd55fcfe42c..cc18147d17a 100644
 +    val v = System.getenv("ENABLE_COMET_SCAN_ONLY")
 +    v != null && v.toBoolean
 +  }
++
++  /**
++   * Whether to enable ansi mode This is only effective when
++   * [[isCometEnabled]] returns true.
++   */
++  protected def enableCometAnsiMode: Boolean = {
++    val v = System.getenv("ENABLE_COMET_ANSI_MODE")
++    v != null && v.toBoolean
```

Comment:
I think this could be simplified as:

```suggestion
+    System.getenv("ENABLE_COMET_ANSI_MODE").exists(_.toBoolean)
```

Reply:
Compiler didn't like the suggested change and I didn't quite understand why. Also, the way it is in the PR, matches the pattern followed by similar other code in the diff, so leaving it as it is.